### PR TITLE
Mentor Page Update

### PR DESCRIPTION
**Pull request by EmmaL789**
Hi Diane, 

I just completed the Mentor Page updating, please let me know if you have any questions. Thanks. 
Emma

### CODE REVIEW

File: src/components/Team.js [hunk 1]
```diff
@@ -841,6 +844,71 @@ function Star(props) {
...
+        </p>
+      </Modal.Body>
+      <Modal.Footer>
+        <Button onClick={props.onHide}>Close</Button>
+      </Modal.Footer>
+    </Modal>
+  );
+}
+
+function Biran(props) {
```
Comment (dianeshan) [inline src/components/Team.js]: typo: `Brian`

File: src/components/Team.js [hunk 2]
```diff
@@ -841,6 +844,71 @@ function Star(props) {
...
+  return (
+    <Modal
+      {...props}
+      size="lg"
+      aria-labelledby="contained-modal-title-vcenter"
+      centered
+    >
+      <Modal.Header closeButton>
+        <Modal.Title id="contained-modal-title-vcenter">
+          Biran Lee
```
Comment (dianeshan) [inline src/components/Team.js]: typo: `Brian`

File: src/components/Team.js [hunk 3]
```diff
@@ -1262,6 +1332,38 @@ const Team = () => {
...
+
+                  <Col sm="12" md="3" className="text-center">
+                    <Button variant="link" onClick={() => setModalShow25(true)}>
+                      <img
+                        className="headshot"
+                        src={biranlee}
+                        alt="Biran's Headshot"
+                      />
+                    </Button>
+                    <h6>Biran Lee</h6>
```
Comment (dianeshan) [inline src/components/Team.js]: typo: `Brian`

File: src/components/Team.js [hunk 4]
```diff
@@ -32,6 +32,9 @@ import tachangfu from "../images/mentors/TaChengFu.png";
 import marcuswhite from "../images/mentors/MarcusWhite.png";
 import shengmingshan from "../images/mentors/ShengmingShan.png";
 import stargao from "../images/mentors/StarGao.png";
+import gloriali from "../images/mentors/GloriaLi.png";
+import biranlee from "../images/mentors/BiranLee.png";
```
Comment (dianeshan) [inline src/components/Team.js]: typo: `brianlee`

File: src/components/Team.js [hunk 5]
```diff
@@ -751,7 +754,7 @@ function Tachang(props) {
     >
       <Modal.Header closeButton>
         <Modal.Title id="contained-modal-title-vcenter">
-          Rong Yue
+          Ta-Chang Fu
```
Comment (dianeshan) [inline src/components/Team.js:757]: good catch!! :)

**Review by dianeshan — CHANGES_REQUESTED** [review]
Overall looks good just some typos on Brian's name

**Review by dianeshan — APPROVED** [review]
looks good!

**Comment by EmmaL789** [comment]
Thank you, Diane. :)

Diane Shan ***@***.***> 于2024年12月13日周五 09:35写道：

**Pull request merged**